components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt

```diff
@@ -263,6 +265,19 @@ open class InlineAutocompleteEditText @JvmOverloads constructor(
         }
     }
 
+    override fun onTextContextMenuItem(id: Int): Boolean {
+        if (id == android.R.id.paste) {
```

Comment:
`<nerd mode>`

Single `return` statement is more preferable, I think. As it's more clear, less error-prone and [mentioned in Kotlin coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#using-conditional-statements).

`</nerd mode>` :smile: